api/v1/helpers/verbs/doFind.js

```diff
@@ -45,11 +45,17 @@ function doFindAndCountAll(reqResNext, props, opts) {
   .then((o) => {
     resultObj.dbTime = new Date() - resultObj.reqStartTime;
     reqResNext.res.set(COUNT_HEADER_NAME, o.count);
+
     const retval = o.rows.map((row) => {
       if (props.modelName === 'Lens') {
         delete row.dataValues.library;
       }
-
+      if ((props.modelName === 'Bots') &&
```

Comment:
On a find, I'd like to see you try to do this in a way which does *not* load all bot library data from the db *then* delete it. That's too expensive.